qiita_db/support_files/patches/14.sql

```diff
@@ -0,0 +1,2 @@
+-- we don't really need an SQL patch, but having nothing here breaks the build
```

Comment:
Yeah. There are a couple of other patches like that. We need something that SQL executable...